torch/_dynamo/output_graph.py

```diff
@@ -1575,6 +1582,13 @@ def add_graph_finalizer(
     ) -> None:
         self.register_finalizer_fns.append(register_finalizer)
 
+    def example_value_from_input_node(self, node: torch.fx.Node):
+        """Extract the non-fake example tensor"""
+        if node.op == "placeholder":
+            return node.meta["grapharg"].example
+        assert node.op == "get_attr"
+        return self.nn_modules[node.target]  # type: ignore[index]
```

Comment:
Is "get_attr" a nn Module specific op?

Reply:
Yeah, dynamo only uses `get_attr` for nn modules (and tensors stored on nn modules).